README.md

```diff
@@ -1 +1,56 @@
-# angular-scope-rootscope-readme
+# $scope and $rootScope
+
+## Overview
+
+## Objectives
+
+- Describe what a $scope is
+- Describe what $rootScope is
+- Describe how Angular utilises Prototypal Inheritance
+- Create a $scope variable
+- Bind the $scope variable to the View
+
+## $scope
+
+You might've noticed the magic of `$scope` in the last lesson - an object that we can assign values to and they appear in our view.
+
+Angular introduces scopes into our application. Whenever we have a new instance of a controller, that controller has a `$scope`. Everything inside our controller can access this scope, however anything outside cannot. `$scope`s can access their parent scope.
```

Comment:
This explanation of what $scope can access is a little bit hard to understand. I like the example graphic below, but it's a lot to take in at once. Can you break this out into chunks? First, show an example of trying to access a property outside of the controller. This can be simple. 

```html
 <div ng-controller="MainController">
    <h2>{{ name }}</h2>
 </div>
 
{{ name }}
 ```

Afterwords, show an example of multiple controllers with separate scopes. Finally, go into the diagram that demos parent scope.